torch/_inductor/fx_passes/post_grad.py

```diff
@@ -795,8 +795,16 @@ def can_inplace(node, mutated_arg):
                     graph.erase_node(copy_node)
 
                 node.target = inplaceable_op.inplace_op
-    for node, replacement in replace_list:
-        node.replace_all_uses_with(replacement)
+
+    def get_replacement(node):
```

Comment:
this is called from one place, maybe just inline it, doesn't need to be a helper function